scene/gui/graph_edit.cpp

```diff
@@ -436,6 +432,8 @@ void GraphEdit::_notification(int p_what) {
 		snap_button->set_icon(get_theme_icon(SNAME("snap")));
 		minimap_button->set_icon(get_theme_icon(SNAME("minimap")));
 		layout_button->set_icon(get_theme_icon(SNAME("layout")));
+
+		zoom_label->set_custom_minimum_size(Size2(48, 0) * get_theme_default_base_scale());
```

Comment:
In general for cases like this I assumed it is okay to update the min size on theme changes as the nodes are internal and unlikely to be touched by users in a very specific way. But this may be a problem for some cases. And this is definitely not going to work what setting up the minimal size of the control node itself like in the ConfirmationDialog's case above.